Pikipick/Pikipick/MultipeerConnectivity/SessionAudience.swift

```diff
@@ -41,16 +41,27 @@ class SessionAudience: NSObject, ObservableObject {
     @Published var connectedPeers: [MCPeerID] = []
     
     // MARK: 이모지 전송
-    /* 이모지 전송
-     - 연결된 peer가 존재한다면, 특정 peer 선택 후 이모지를 전송합니다
-     */
-    func send(emoji: EmojiName, receiver: MCPeerID) {
-        log.info("sendEmoji: \(String(describing: emoji)) to \(receiver.displayName)")
-
+    func sendEmoji(sendEmoji: String, receiver: MCPeerID) {
+        log.info("sendEmoji: \(String(describing: sendEmoji)) to \(receiver.displayName)")
+        let sendData = UUID().uuidString + sendEmoji
+        // Is there any Connected Peers more than 1
+        if !session.connectedPeers.isEmpty {
+            do {
+                try session.send(sendData.data(using: .utf8)!, toPeers: [receiver], with: .reliable)
+            } catch {
+                log.error("Error for sending: \(String(describing: error))")
+            }
+        }
```

Comment:
```suggestion
        guard session.connectedPeers.isEmpty else { return }
        do {
            try session.send(sendData.data(using: .utf8)!, toPeers: [receiver], with: .reliable)
        } catch {
            log.error("Error for sending: \(String(describing: error))")
        }
```

guard문을 쓰면 인덴트를 줄일 수 있답니당

Reply:
감사합니다! 반영했습니다🔥🔥🔥